_installation/single-multiple.md

```diff
@@ -26,16 +33,59 @@ CREATE USER babelfish_user WITH CREATEDB
 	CREATEROLE PASSWORD '<PUT_SECRET_PASS_HERE>' INHERIT;
 DROP DATABASE IF EXISTS demo;
 
+/* the Babelfish database */
 CREATE DATABASE demo OWNER babelfish_user;
+ALTER SYSTEM SET babelfishpg_tsql.database_name = 'demo';
+SELECT pg_reload_conf();
+
+/* determine the migration mode */
+ALTER DATABASE demo SET babelfishpg_tsql.migration_mode = 'single-db'|'multi-db';
 
 \c demo
 
 CREATE EXTENSION IF NOT EXISTS "babelfishpg_tds" CASCADE;
-ALTER SYSTEM SET babelfishpg_tsql.database_name = 'demo';
-ALTER DATABASE demo SET babelfishpg_tsql.migration_mode = 'single-db'|'multi-db';
 CALL SYS.INITIALIZE_BABELFISH('babelfish_user');
 ```
 
-The `ALTER DATABASE` statement assigns the correct parameter to the database.
-The init function will then perform all the steps needed to perform the setup.
+The `ALTER DATABASE` statement assigns the correct migration mode to the
+database.  The `SYS.INITIALIZE_BABELFISH` function will then perform all
+the steps needed to set up Babelfish.
+
+Once you are connected to the database via TDS, you can utilize the `USE`
+command to select the current database as you would do it in Microsoft
+SQL Server (with `single-db`, you can have only a single database).
+
+
+### Choosing a migration mode
+
+Each migration mode has advantages and disadvantages. Choose your migration
+mode based on the number of user databases you have, and your migration plans.
+After you create a cluster for use with Babelfish, you can't change the
+migration mode. When choosing a migration mode, consider the requirements of
+your user databases and clients.
+
+When you initialize Babelfish, Babelfish creates the system databases
+`master`. and `tempdb`.  If you created or modified objects in the
+system databases (`master` or `tempdb`), make sure to re-create those objects
+in your new cluster.  Unlike SQL Server, Babelfish doesn't reinitialize `tempdb`
+after a cluster reboot.
+
+Use single database migration mode in the following cases:
+
+- If you are migrating a single SQL Server database. In single database mode,
+  migrated schema names are identical to the original SQL Server schema names.
+  When you migrate your application, you need to make fewer changes to your
+  SQL code.
```

Comment:
this is not correctly worded. "When you migrate your application, you need to make fewer changes ..." should be "When you ultimately migrate your application to native PostgreSQL, you need to make fewer changes..."

Reply:
Agreed. Changed to:
"In case you plan to ultimately migrate your application to native
  PostgreSQL, your SQL code will require fewer modifications."